### PR TITLE
Fixes autocorrect "[ :abc, :123]" to "%i[abc 123]"

### LINKED ISSUE #2: %i with arrays of symbols
``` ruby
[:one, :two, :three]
```
gets changed to 

``` ruby
%i[one, two, three]
```
I don't think we like this. @cam156

### PR DESCRIPTION
**Pull request by carolyncole**
Fixes #2

### CODE REVIEW

File: style/ruby/.rubocop.yml [hunk 1]
```diff
@@ -467,6 +467,9 @@ Style/InitialIndentation:
   Description: >-
     Checks the indentation of the first non-blank non-comment line in a file.
   Enabled: false
+  
+Style/SymbolArray:
```
Comment (awead) [inline style/ruby/.rubocop.yml:474]: @cam156 can you add a `Description:` here that describes what it does?

**Review by awead — CHANGES_REQUESTED** [review]

File: style/ruby/.rubocop.yml (hunk 1 above)
Comment (carolyncole) [reply style/ruby/.rubocop.yml:474]: @awead where are you getting the descriptions from?

Comment (awead) [reply style/ruby/.rubocop.yml:474]: Sometimes from the documentation, but what you wrote in the commit is fine: `changes [ :abc, :123] to %i[abc 123]`

Comment (carolyncole) [reply style/ruby/.rubocop.yml:474]: @awead but that is not the official description for the Linter.  Where are those coming from?  From what I can see your descriptions do not match the comments in the code.  I do not think making stuff up is wise.

Comment (awead) [reply style/ruby/.rubocop.yml:474]: I would just use http://www.rubydoc.info/github/bbatsov/rubocop/master/Rubocop/Cop/Style/SymbolArray

Comment (carolyncole) [reply style/ruby/.rubocop.yml:474]: @awead updated the cop with a description and based on the code and added a link to the location of the documentation

Comment (awead) [reply style/ruby/.rubocop.yml:474]: @cam156 awesome!

**Pull request merged**